apps/examples/testcase/ta_tc/media/utc/utc_media_speechdetector.cpp

```diff
@@ -1,6 +1,6 @@
 /****************************************************************************
  *
- * Copyright 2018 Samsung Electronics All Rights Reserved.
+ * Copyright 2024 Samsung Electronics All Rights Reserved.
```

Comment:
you should not change year. we pushed this file at 2018

Reply:
Thanks. 
I just updated from Media Repo.
It is fixed now.